diff --git a/node_modules/react-dropzone-uploader/dist/Dropzone.tsx b/node_modules/react-dropzone-uploader/dist/Dropzone.tsx
index da29a80..4bfe8c2 100644
--- a/node_modules/react-dropzone-uploader/dist/Dropzone.tsx
+++ b/node_modules/react-dropzone-uploader/dist/Dropzone.tsx
@@ -228,6 +228,7 @@ export interface IDropzoneProps {
   classNames: IStyleCustomization<string>
   styles: IStyleCustomization<React.CSSProperties>
   addClassNames: IStyleCustomization<string>
+  imageState: object
 
   /* component injection */
   LayoutComponent?: ReactComponent<ILayoutProps>
@@ -500,7 +501,7 @@ class Dropzone extends React.Component<IDropzoneProps, { active: boolean; dragge
     let params: IUploadParams | null = null
     try {
       params = await getUploadParams(fileWithMeta)
-    } catch (e) {
+    } catch (e: any) {
       console.error('Error Upload Params', e.stack)
     }
     if (params === null) return
@@ -754,6 +755,7 @@ Dropzone.defaultProps = {
   classNames: {},
   styles: {},
   addClassNames: {},
+  imageState: {}
 }
 
 // @ts-ignore
